helpdesk_mgmt/models/helpdesk_ticket_stage.py

```diff
@@ -14,6 +14,7 @@ class HelpdeskTicketStage(models.Model):
         string='Unattended')
     closed = fields.Boolean(
         string='Closed')
+    portal_user_can_close = fields.Boolean()
```

Comment:
could you use this boolean to disallow [any logged in user to close any ticket](https://github.com/OCA/helpdesk/blob/44b501023dc4b4f06203b72be8fbfc0a16b7e64d/helpdesk_mgmt/controllers/main.py#L20)?

Reply:
Done, thanks